overlord/snapstate/autorefresh.go

```diff
@@ -482,12 +482,17 @@ func getTime(st *state.State, timeKey string) (time.Time, error) {
 // that period the refresh will go ahead despite application activity.
 func inhibitRefresh(st *state.State, snapst *SnapState, info *snap.Info, checker func(*snap.Info) error) error {
 	if err := checker(info); err != nil {
+		days := int(maxInhibition.Truncate(time.Hour).Hours() / 24)
 		now := time.Now()
 		if snapst.RefreshInhibitedTime == nil {
 			// Store the instant when the snap was first inhibited.
 			// This is reset to nil on successful refresh.
 			snapst.RefreshInhibitedTime = &now
 			Set(st, info.InstanceName(), snapst)
+			if _, ok := err.(*BusySnapError); ok {
+				st.Warnf(i18n.NG("refresh of snap %q postponed for up to %d day because it is in use",
+					"refresh of snap %q postponed for up to %d days because it is in use", days), info.SnapName(), days)
```

Comment:
I'm still thinking about this, but what about something more like:
`snap %q is currently active. Its refresh will be postponed for up to %d days to wait for the snap to become inactive.`

Reply:
I wanted to avoid the word "active" because IIRC it has some semantics in the snapd stack but I think that this is one of the user-facing messages that just ought to be readable without any extra glossary, it should just make sense.

I'll happily merge this suggestion after we are done with the technical review.

Reply:
what about just 
```
snap %q is currently in use. Its refresh will be postponed for up to %d days to wait for the snap to no longer be in use.
```
?

Reply:
Applied, thank you!